libs/filamat/src/MaterialBuilder.cpp

```diff
@@ -212,6 +213,53 @@ MaterialBuilder& MaterialBuilder::parameter(const char* name, SamplerType sample
     return parameter(name, samplerType, SamplerFormat::FLOAT, precision);
 }
 
+template<typename T, typename>
+MaterialBuilder& MaterialBuilder::constant(const char* name, ConstantType type, T defaultValue) {
```

Comment:
`MaterialBuilder.h` already make plenty of uses of the STL, so I would just use `std::string_view` for the `name` parameter.

Reply:
I'm not sure it makes sense to use `string_view` here either. We need to make a copy of the string anyway, plus all the other methods on `MaterialBuilder` are using a `const char*`.